CONTRIBUTING.md

```diff
@@ -41,90 +41,95 @@ Unit tests are nice for atomic components, but since jdbi is a complex ecosystem
 
 Since our tests essentially describe and verify jdbi's behavior, changing their specifics where it isn't inherently necessary is considered a red flag.
 
+The use of mocks and mocking framework is generally discouraged. There are a number of existing tests that use Mockito and they are a pain to maintain.
+
 ## Pull requests
 
-Expect your pull request to be scrutinized in even the tiniest details (down to grammar in javadoc), and to need to address many remarks even for small changes. We strive for a healthy balance between subjective perfection and practical considerations, but we are firmly against doing a quick and sloppy job that will require a lot of follow-up work later.
+We strive for a healthy balance between subjective perfection and practical considerations, but we are firmly against doing a quick and sloppy job that will require a lot of follow-up work later.
```

Comment:
We could add a word or two about meaningful PR titles and commit messages.

Reply:
Personally, I don't care much about ceremony here. If someone sends a PR with "fix" as the commit message, we will call it out. But I want people to focus on code and functionality, not ceremony. We are a pretty informal bunch anyway. ;-)